x/staking/types/validator.go

```diff
@@ -117,7 +117,11 @@ type ValidatorsByVotingPower []Validator
 func (valz ValidatorsByVotingPower) Len() int { return len(valz) }
 
 func (valz ValidatorsByVotingPower) Less(i, j int) bool {
-	if valz[i].ConsensusPower() == valz[j].ConsensusPower() {
+	return valz.LessAfterPowerReductionApply(i, j, sdk.DefaultPowerReduction)
```

Comment:
This shouldn't use DefaultPowerReduction.  Either it should use no power reduction or preferably be removed altogether (i.e. require that a powerreduction is passed in).

If it uses Default, I think it will lead to bugs where people accidentally use it, and don't realize it is only correct for when using Default.

Reply:
I also think that in all cases where two validators are ordered, they should be ordered by their unreduced power

Reply:
No, I agree with @antstalepresh, for this I think we should sort in the same order as Tendermint does.

https://github.com/sikkatech/cosmos-sdk/pull/65#discussion_r566550144

Reply:
Ah that makes sense. I wonder if long term we can adapt Tendermint to take in more precision.